### ISSUE TITLE
🧹 Chore: Move Legacy `travel-model-two` Documentation

### ISSUE THREAD

**Issue by DavidOry** [BayAreaMetro/tm2py#148]
The current `travel-model-two` repository contains everything necessary to run TM2.1 and earlier, including the Java source code, application scripts, and configuration files. For TM2.2 and beyond, the model controller infrastructure lives in `tm2py`, scenario-specific configuration files will live in `travel-model-two-configs`, and the Java resident passenger component software will live in `travel-model-two` (though we may want to rename to `travel-model-two-resident-passenger` to make it clear what it is.

Right now, the legacy `travel-model-two` repository contains a host of useful, but not up to date information. Some of this information, e.g., [process](https://bayareametro.github.io/travel-model-two/develop/process/), is out of date and replaced by `tm2py`. Others, e.g., [geographies](https://bayareametro.github.io/travel-model-two/develop/geographies/), remains useful and should continue to live somewhere. 

Progress:

- [ ] Sufficiently defined
- [ ] Approach decided
- [ ] Implemented

## Considerations
The question is where should this documentation be moved to. @lmz, @gregerhardt: any thoughts? My initial thought is to move it to `tm2py`, so that web presence becomes a one-stop shop.

A similar question for where you want to put things like the MAZ and TAZ shape files. To me these should be in the `configs` directory. Thoughts?

**Comment by DavidOry** [comment]
> [@i-am-sijia](https://github.com/i-am-sijia) , [@DavidOry](https://github.com/DavidOry) - can we move this up in priority- like start this month- and just move to tm2py? I can do the initial move, but I don't know what's changed or needs to be updated. I'm really having trouble keeping track of where info is across Github, box, asana, and slack. Not sure a better way yet. cc: [@lmz](https://github.com/lmz) , [@gregerhardt](https://github.com/gregerhardt)

Yes. We have a small amount of "as-needed" support budget in the current task order. Perhaps we could have a call to go through the current `travel-model-two` content and identify which pages are relevant and which are obsolete?

**Comment by Ennazus** [comment]
I think I moved things well enough for now to close the issue.
